framework/vue/link.ts

```diff
@@ -0,0 +1,22 @@
+import { useRouter } from "./router.ts";
+import { defineComponent, h } from "vue";
+
+export const Link = defineComponent({
+  name: "Link",
+  props: {
+    to: {
+      type: String,
+      default: "",
+    },
+  },
+  setup() {
+    const { url, params } = useRouter();
+    return {
+      url,
+      params,
+    };
+  },
+  render() {
+    return h("a", { href: this.$props.to }, this.$slots.default ? this.$slots.default() : []);
```

Comment:
seems the `Link` component doesn't use the [redirect.ts](https://github.com/alephjs/aleph.js/blob/a5af6856dc3bc3d10b35c1b7b1204445045ddfec/framework/core/redirect.ts) of framework core. In client, when you click a link, the page should not be refreshed, you need to import the route modules and fetch route data, then use the `redirect` function to redirect page

Reply:
or you want to implement it in other pr?

Reply:
> or you want to implement it in other pr?

Sorry, I didn't reply in time, I was coding for this feature, it was done right now.

Reply:
> seems the `Link` component doesn't use the [redirect.ts](https://github.com/alephjs/aleph.js/blob/a5af6856dc3bc3d10b35c1b7b1204445045ddfec/framework/core/redirect.ts) of framework core. In client, when you click a link, the page should not be refreshed, you need to import the route modules and fetch route data, then use the `redirect` function to redirect page

Done